pkg/cni/common.go

```diff
@@ -45,15 +48,21 @@ const (
 )
 
 // Prepare prepares the CNI deployment
-func Prepare() error {
-	if _, err := os.Stat(DefaultCNIConflist); os.IsNotExist(err) {
-		glog.V(1).Infof("[kubic] creating default configuration for CNI in '%s'", DefaultCNIConflist)
+func Prepare(cfg *config.KubicInitConfiguration) error {
+	cniConflist := strings.Join([]string{config.DefaultCniConfDir, DefaultCNIConfName}, "")
```

Comment:
Better to use the [path.Join](https://golang.org/pkg/path/#Join)

```suggestion
	cniConflist := path.Join(config.DefaultCniConfDir, DefaultCNIConfName)
```
and you should remove the starting `/` in `DefaultCNIConfName`

Reply:
@inercia Path.Join is really clean I hate to use []string{}, thanks.